src/annotator/util/observable.js

```diff
@@ -58,6 +58,7 @@ export function delay(delay, src) {
  * Buffers events from a source Observable, waiting for a pause of `delay`
  * ms with no events before emitting the last value from `src`.
  *
+ * @template T
```

Comment:
I can see how this function takes and returns an `Observable`, but I'm having trouble grokking how `<T>` flows through to the returned `Observable`. Anything you can do to shed some light?